tools-local/tasks/project.json

```diff
@@ -2,7 +2,7 @@
   "dependencies": {
     "System.Reflection.Metadata": "1.3.0",
     "Newtonsoft.Json": "9.0.1",
-    "NuGet.ProjectModel": "3.5.0",
+    "NuGet.ProjectModel": "4.3.0-preview1-4051",
```

Comment:
FYI - I needed to move to the latest NuGet here because 4.3.0 depends on newtonsoft 9 for all TFMs.  Earlier versions of NuGet had `net45` depending on newtonsoft 6, which caused me some problems.